app/components/certificate/panels/node.tsx

```diff
@@ -0,0 +1,45 @@
+import { Divider, TabPanel, Text, Link } from '@chakra-ui/react';
+import CertificateDisplay from '../certificate-display';
+
+interface NodePanelProps {
+  publicKey: string;
+  privateKey: string;
+}
+
+export default function NodePanel({ publicKey, privateKey }: NodePanelProps) {
+  return (
+    <TabPanel>
+      <Text marginBottom={4}>
+        You can use your certificate with{' '}
+        <Link
+          href="https://nodejs.org/api/https.html#httpscreateserveroptions-requestlistener"
+          isExternal
+        >
+          node.js to create a secure HTTPS server
```

Comment:
I think we could capitalize the first letter to match the [registered name](https://nodejs.org/en).

Reply:
OK, done.